### PR TITLE
PE-714: Basic README examples for integrating with ArDrive Core

### PR DESCRIPTION
**Pull request by fedellen**
This PR adds a few basic examples of how to interact with ArDrive Core JS as a dependency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,90 @@ ArDrive Core is a TypeScript library that contains the essential back end applic
...
+
+// Wrap folder and all of its contents for upload
+const wrappedFolder = wrapFileOrFolder('path/to/folder');
+
+// Upload a private folder and all its contents
+const uploadFileResult = await arDrive.uploadAllEntities({
+    entitiesToUpload: [
+        {
+            wrappedEntity: wrappedFolder,
+            destFolderId: EID('76543214a-eb5e-4134-8ae2-a3946a428ec7'),
```
[the remark this answers is not in the document]
Comment (karlprieb) [reply README.md:79]: I don't think that I agree with that. Differently from `EntityKey` example where we're not validating the data here with `EID` we guarantee that the data is valid.
For `wrappedEntity` a string can work, but we will need to understand if we really want that complexity at `uploadAllEntities`

**Review by karlprieb — APPROVED** [review]
LGTM 👍 
This addition is great!

File: README.md [hunk 2]
```diff
@@ -4,6 +4,90 @@ ArDrive Core is a TypeScript library that contains the essential back end applic
...
+    folderName: 'My New Private Folder',
+    driveKey,
+    parentFolderId: EID('47162534a-eb5e-4134-8ae2-a3946a428ec7')
+});
+```
+
+```ts
+import { wrapFileOrFolder, EntityKey, EID } from 'ardrive-core-js';
+
+// Derive a private drive key from raw drive key string
```
[the remark this answers is not in the document]
Comment (matibat) [reply README.md:68]: I agree. There could be an `EntityKey::from(keyAsBase64: string)`

**Review by matibat — APPROVED** [review]
:100:

**Pull request merged**